dist/Renderer.js

```diff
@@ -0,0 +1,13 @@
+
+class Renderer{
+     
+     renderData (allCityData) {
```

Comment:
`renderCities()` or `renderWeather()`
because at some point you will be working with more sets of data and more complex data. It is better to be specific